build-tools/automation/yaml-templates/stage-msbuild-emulator-tests.yaml

```diff
@@ -11,6 +11,8 @@ parameters:
   repositoryAlias: 'self'
   commit: ''
   usesCleanImages: true
+  shouldFailOnIssue: true
```

Comment:
It would be nice if the PR description explained *what* it is trying to accomplish with these 2 variables, as I don't know what they're for.